foundry.toml

```diff
@@ -9,6 +9,7 @@ remappings = [
     "@openzeppelin-contracts-5.2.0/=dependencies/@openzeppelin-contracts-5.2.0/",
     "forge-std-1.9.6/=dependencies/forge-std-1.9.6/",
 ]
+fs_permissions = [{ access = "read-write", path = "out/"}]
```

Comment:
I would change this to `path = "deployments/"` because `out` is the directory that Forge uses for its artifacts. Better not to avoid conflict.